homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -26,20 +26,24 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
     all_devices = api.get_all_devices()
     for dev_id, device_properties in all_devices.items():
-        if device_properties["class"] == "heater_central":
-            data = api.get_device_data(dev_id)
-            for binary_sensor, dummy in BINARY_SENSOR_MAP.items():
-                if binary_sensor in data:
-                    entities.append(
-                        PwBinarySensor(
-                            api,
-                            coordinator,
-                            device_properties["name"],
-                            binary_sensor,
-                            dev_id,
-                            device_properties["class"],
-                        )
-                    )
+        if "heater_central" not in device_properties["class"]:
```

Comment:
Are there more than one item in `device_properties["class"]`? The old check just checked for equality not membership.

Reply:
You are right, should check for non-equality, but this is what you meant with your comment on the other PR, right? I'll further clean up all platforms than later today.

Reply:
Yes, the concept is correct here. :+1:

Reply:
Will work on that later today

Reply:
I think this is all I could find, there is (in `climate.py` an if/else construction in `def async_set_temperature` but I'm not sure if guarding would make it easier to read a double condition statement. (Just thinking out loud)

Testing on small dev-setup seems to work, also co-testing this on our custom_component now.